### PR TITLE
mcsuchtifyde

### PR DESCRIPTION
**Pull request by Vinnipapi**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by toohard2explain** [comment]
please rename your pull request

**Comment by toohard2explain** [comment]
also, is the server even accessible?

image

**Comment by Vinnipapi** [comment]
> also, is the server even accessible?
> 
> image

Yes at the moment we are undergoing maintenance and due to holiday break we can not work on our network so basicly it will be reaccessable in early 2024

**Comment by Vinnipapi** [comment]
> please rename your pull request

To what? I renamed it to the current server ip if its alright!

File: minecraft_servers/suchtify/manifest.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "server_wildcards": [
+    "%.suchtify.de"
+  ],  
+
+  "supported_languages": [
+     "de"
+  ],  
+
+  "social": {
+    "web": "https://suchtify.de",
```
Comment (DoJapHD) [inline minecraft_servers/suchtify/manifest.json:14]: Website not reachable

File: minecraft_servers/suchtify/manifest.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "web_shop": "https://suchtify.de/shop",
```
Comment (DoJapHD) [inline minecraft_servers/suchtify/manifest.json:15]: Website not reachable

File: minecraft_servers/suchtify/manifest.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    "web_support": "https://wiki.suchtify.de",
```
Comment (DoJapHD) [inline minecraft_servers/suchtify/manifest.json:16]: Website not reachable

File: minecraft_servers/suchtify/manifest.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    "twitter": "SuchtifyDE",
+    "instagram": "suchtifyde",
+    "tiktok": "suchtifyde"
+  },
+  
+  "gamemodes": {
+    "oneblock": {
+      "name": "OneBlock", 
+      "color": "#004f12",
+      "url": "https://suchtify.de",
```
Comment (DoJapHD) [inline minecraft_servers/suchtify/manifest.json:28]: Doesn't make sense here, remove

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server in general not reachable; reopen PR once reachable

**Pull request closed without merging**